frontend/src/Pages/StudentManagement.tsx

```diff
@@ -147,8 +148,8 @@ export default function StudentManagement() {
                             enumType={{
                                 'Name (A-Z)': 'name_last asc',
                                 'Name (Z-A)': 'name_last desc',
-                                'Account Created ↓ ': 'created_at desc',
-                                'Account Created ↑ ': 'created_at asc'
+                                'Account Created v ': 'created_at desc',
```

Comment:
See my other comment about AdminManagement. We should be using the icons and not `v` or `^`